public/views/index.html

```diff
@@ -240,7 +240,7 @@ <h3>XOXO</h3>
 
 <!-- Footer Start -->
 <section class="foot">
-  <div><small class="legal">Legal stuff: This game is based on the <a href="http://cardsagainsthumanity.com">Cards Against Humanity</a> card game, which is CC licensed (BY-NC-SA). Cards for Humanity is not affiliated with Cards Against Humanity. In complying with the Creative Commons license of the Cards Against Humanity card game, all proceeds from donations go directly to charity, as managed by <a href="http://crowdrise.com">crowdrise</a>.</small></div>
+  <div><small class="legal">Legal stuff: This game is based on the <a href="http://localhost:3000">Cards Against Humanity</a> card game, which is CC licensed (BY-NC-SA). Cards for Humanity is not affiliated with Cards Against Humanity. In complying with the Creative Commons license of the Cards Against Humanity card game, all proceeds from donations go directly to charity, as managed by <a href="http://crowdrise.com">crowdrise</a>.</small></div>
```

Comment:
This is the license so I believe it should actually link to the cardsagainsthumanity site

Reply:
I have implemented the feedback